internal/sdk/cloudian/sdk_test.go

```diff
@@ -1,16 +1,57 @@
 package cloudian
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"math/rand"
+	"net/http"
+	"net/http/httptest"
 	"reflect"
 	"strings"
 	"testing"
 	"testing/quick"
+
+	"github.com/google/go-cmp/cmp"
 )
 
+func TestGetGroup(t *testing.T) {
+	expected := Group{
+		GroupID: "QA",
+	}
+
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(`{"groupId": "QA"}`))
```

Comment:
```suggestion
		w.Write([]byte(`{"groupId": "QA"}`))
```

	// If WriteHeader is not called explicitly, the first call to Write
	// will trigger an implicit WriteHeader(http.StatusOK).